dev/widgets.py

```diff
@@ -66,4 +104,108 @@ def set_custom_style(self, style):
         self.set_style(lv.line.STYLE.MAIN, style)
 
     def set_line_points(self, points):
-        self.set_points(points, len(points))
+        formatted = self.format(points)
+        self.set_points(formatted, len(formatted))
+
+
+class Bar(lv.bar):
+
+    def __init__(self, parent, x, y, width=None, height=None):
+        # Note that for the 1.8" display, height must be a minimum of 11
+        super().__init__(parent)
+        self.x = x
+        self.y = y
+        self.width = width
+        self.height = height if height >= 11 else 11
+        self.set_size(self.width, self.height)
+        self.set_pos(x, y)
+
+    def set_animation_time(self, time):
+        self.set_anim_time(time)
+
+    def set_value_animation(self, value, anim):
+        # Set anim=True to have an animation and False for static
```

Comment:
Similar to other comment use `""" """` instead of inline comment `#`:
```python
def set_value_animation(self, value, anim):
    """ Set anim=True ...
    """
```